_includes/aip-list-item.html

```diff
@@ -0,0 +1,4 @@
+{% comment %} This file is used to create a navigation element to AIPs in a navigation menu. When this file is included it *must* be sent with a `category` and a `menu` value. The `category` value contains the page a link is being created for. The `menu` value represents the `nav-menu` object that defines the navigation menu.{% endcomment %}
+<a href="{{ include.category.url | relative_url }}">
+  <span class="aip-number">{{ include.category[include.menu.frontMatter][include.menu.id] }}</span> {{ include.category.title }}
+</a>
```

Comment:
AIPs headers in the nav menu need to be rendered differently than AIP categories. So I had to write a custom includes that is part of the `_data/nav.yml` configuration (ln 17) so you can optionally specify how you want the line rendered.